modules/tile-converter/src/i3s-converter/i3s-converter.ts

```diff
@@ -28,10 +28,30 @@ import {LAYERS as layersTemplate} from './json-templates/layers';
 import {NODE as nodeTemplate} from './json-templates/node';
 import {SHARED_RESOURCES_TEMPLATE} from './json-templates/shared-resources';
 import {validateNodeBoundingVolumes} from './helpers/node-debug';
-import type {SceneLayer3D} from '../types';
-import type {GLTFMaterial} from '@loaders.gl/gltf';
 import {GeoidHeightModel} from '../lib/geoid-height-model';
 import {Tileset3DProps} from '@loaders.gl/tiles';
+import TileHeader from '@loaders.gl/tiles/src/tileset/tile-3d';
+
+import type {GLTFMaterial} from '@loaders.gl/gltf';
```

Comment:
Nit: I put all the `import type` statements first. Their semantics are different (they are not included in generated code).

Reply:
Done